src/app/components/autocomplete/autocomplete.component.html

```diff
@@ -0,0 +1,25 @@
+<form>
+	<mat-form-field appearance="outline" onsubmit="return false">
```

Comment:
`onsubmit="return false"` should be on the form. It's used to prevent page reloads on submit

Reply:
yeah, this was a mistake, I overlooked it

Reply:
fixed in the new commit